tests/unit/core/server/test_server.js

```diff
@@ -71,11 +72,14 @@ export class ServerTestHelper {
     global.webpackIsomorphicTools = {
       assets: () => fakeAssets,
     };
+    // We don't want the `document` object on the server.
+    delete global.document;
```

Comment:
Good idea. Maybe delete `window`, too?

Reply:
I deleted `document` mainly because `universal-cookie` uses it to determine if it is in server or client mode.